tests/foreman/cli/test_jobtemplate.py

```diff
@@ -21,190 +21,156 @@
 
 from robottelo import ssh
 from robottelo.cli.base import CLIReturnCodeError
-from robottelo.cli.defaults import Defaults
 from robottelo.cli.factory import CLIFactoryError
 from robottelo.cli.factory import make_job_template
-from robottelo.cli.factory import make_location
-from robottelo.cli.factory import make_org
 from robottelo.cli.job_template import JobTemplate
 from robottelo.datafactory import invalid_values_list
-from robottelo.test import CLITestCase
+from robottelo.datafactory import parametrized
 
 TEMPLATE_FILE = 'template_file.txt'
 TEMPLATE_FILE_EMPTY = 'template_file_empty.txt'
 
 
-class JobTemplateTestCase(CLITestCase):
-    """Implements job template tests in CLI."""
+@pytest.fixture(scope="module")
+def module_template():
```

Comment:
You've moved the `ssh` commands into a fixture, but the fixture isn't being used by any of the tests. If it just needs to run once for all the tests, you could use `pytestmark` at the top of the module:

`pytestmark = [pytest.mark.usefixtures('module_template')]`

Reply:
This is a required change, @pondrejk please add the `usefixtures` mark on the module as described, or I believe `autouse=True` would work well here too.